index.html

```diff
@@ -46,7 +46,7 @@ <h1>My Recent Works</h1>
         <li class="item">
           <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
           <div class="text-card">
-            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <h2 class="subtitle">Multi-Post Stories Gain+Glory</h2>
             <ul class="courses">
               <li>Ruby on Rails</li>
               <li>CSS</li>
```

Comment:
- [ ] there are two contrast tests that are failings instead of  2.6:1 points you should get at least 4.5:1 score for a best accessibility experience, you can run this test [here](https://color.a11y.com/) by putting your website link , kindly try to fix those colors pairs for a better score

![image](https://user-images.githubusercontent.com/89282221/187504654-e7bb7ee9-984a-4313-8e90-4e9a8064b260.png)